extensions/dub-link-shortener/src/api/oauth.ts

```diff
@@ -0,0 +1,36 @@
+import { OAuth } from "@raycast/api";
+import { OAuthService } from "@raycast/utils";
+import { Dub } from "dub";
+
+let dub: Dub | null = null;
+
+const client = new OAuth.PKCEClient({
+  redirectMethod: OAuth.RedirectMethod.Web,
+  providerName: "Dub",
+  providerIcon: "command-icon.png",
+  providerId: "dub",
+  description: "Connect your Dub account",
+});
+
+export const dubOAuth = new OAuthService({
+  client,
+  clientId: "dub_app_d70ed8f23a7eb32a0532776e5de0be08796cc82e12a3c806",
+  scope: ["links.write", "tags.write", "workspaces.read", "domains.read"].join(" "),
```

Comment:
Oh gotcha, sorry I misunderstood, thanks for clarifying! 🙏 

@pernielsentikaer yes, we've added all the necessary scopes and also included `tags.write` for now if we do want to add the ability to create/update tags in the future: